cloudfoundry-incubator/cf-abacus
# Restructure tests

Restructure introduces:
* **test\integration** folder:
  * contains integration tests which may depend on local MongoDB or RabbitMQ
  * run via `yarn run integration`
* **test\scenario** folder:
  * contains end-to-end scenarios using a fully deployed system
  * run via `yarn run scenario`
* **test\performance** folder:
  * contains load tests and require a pre-deployed system
  * run via `yarn run performance`
* **test\dependency** folder:
  * contains tests against remote systems (ones that Abacus depends on) and verify that their API contracts have not changed
  * run via `yarn run dependency`
